src/controllers/pageController.js

```diff
@@ -0,0 +1,101 @@
+const asyncHandler = require("express-async-handler");
+const Location = require("../models/locationModel");
+
+// @desc Get pages
+// @route GET /api/page
+// FOR PAGE /api/page?pageNumber=1
+// PAGE with SEARCH and CATEGORY example /api/page?pageNumber=1&search=shopping&category=Attractions
+// can be varied as needed
+const pages = asyncHandler(async (req, res) => {
+  const locationsPerPage = 10;
+  let pageNumber = req.query.pageNumber - 1 || 0;
```

Comment:
What's the purpose of `|| 0` here?

Reply:
If I won’t get a page number from the front end I will use page 0 (1) as standard. I think that code will do this or not?